src/components/axis.js

```diff
@@ -1,4 +1,21 @@
-export default function axis() {
+export default function axis(layout, flags, theme) {
+  const axisComponent = flags.isEnabled('CLIENT_IM_2022')
+    ? (layout.components || []).find((c) => c.key === 'axis')
+    : {};
+  const labelStyle =
+    axisComponent && axisComponent.axis && axisComponent.axis.label && axisComponent.axis.label.name
+      ? {
+          settings: {
+            labels: {
+              fontSize: axisComponent.axis.label.name.fontSize,
+              fontFamily: axisComponent.axis.label.name.fontFamily,
+              fill: axisComponent.axis.label.name.fontColor
+                ? axisComponent.axis.label.name.fontColor.color
+                : theme.getStyle('object.mekkochart', 'axis.label.name.color', 'color'),
```

Comment:
Does `object.mekkochart` actually exist in theme or where did you get that from?

Reply:
So what will `theme.getStyle('object.mekkochart', 'axis.label.name.color', 'color')` return if `mekkochart` is not defined in the theme?

Reply:
I think that there are default theme values and that is what other charts are following, for example charts from https://github.com/qlik-oss/dist-flow 
I am not completely sure how sn-mekko-chart is setup with regards to this.

Is there a way in sn-mekko-chart to get default colours if nothing is specified?

Reply:
The defaults depend on how which paths you send. I don't remember how it worked. I think @LiKang6688 knows this

Reply:
The charts you linked to all exists as custom theme properties ([see docs](https://help.qlik.com/en-US/sense-developer/November2021/Subsystems/Extensions/Content/Sense_Extensions/CustomThemes/custom-themes-properties.htm)), so for them it makes sense.

But as the mekko chart does not. It might get some default value out of `getStyle`, but you need to know that, before using it.

If you don't get reasonable value out of it. Maybe you can use the chart generic value:
`theme.getStyle('object', 'label.value', 'color'); // or something like that`

Reply:
I get these values by default

`theme.getStyle('object.mekkochart', 'label.value.color', 'color')` `#333333`
`theme.getStyle('object', 'label.value', 'color')` `#333333`
`theme.getStyle('object', 'axis.label', 'color')` `#404040`
`theme.getStyle('object.mekkochart', 'axis.label.name.color', 'color')` `#595959`

I have not enabled any feature flags and was just trying to console.log the default values.

Reply:
`theme.getStyle('object', 'label.value', 'color') #333333`
`theme.getStyle('object', 'axis.label', 'color') #404040`

Seems like the way to go for me. Than it will get the default colors that is generic for all charts. If we do end up adding support for the mekko chart in theme. We can change it here than.